src/views/FindOrder.vue

```diff
@@ -104,61 +104,80 @@
 
               <div class="metadata">
-                <ion-note>Ordered on 7 Jan 2021</ion-note>
-                <ion-badge>Approved</ion-badge>
+                <ion-note> {{ $t("Ordered on") }} {{ $filters.formatUtcDate(order.doclist.docs[0].orderDate, 'YYYY-MM-DDTHH:mm:ssZ') }} </ion-note>
+                <ion-badge> {{ order.doclist.docs[0].orderStatusId }} </ion-badge>
               </div>
             </section>
 
-            <section class="section-grid" @click="() => router.push('/order')">
-              <div v-for="item in 7" :key="item">
+            <section class="section-grid" @click="() => router.push(`/order/${order.doclist.docs[0].orderId}`)">
+              <div v-for="(item, index) in order.doclist.docs" :key="index" :item="item">
                 <ion-card>
                   <ion-item>
                     <ion-thumbnail slot="start">
-                      <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+                      <Image :src="getProduct(item.productId).mainImageUrl" />
                     </ion-thumbnail>
                     <ion-label>
-                      <p>Brand</p>
-                      Virtual Name
-                      <p>{{ $t("Color") }} : color</p>
-                      <p>{{ $t("Size") }}: size</p>
+                      <p> {{ getProduct(item.productId).brandName ? getProduct(item.productId).brandName : '-' }} </p>
+                      {{ item.virtualProductName }}
+                      <p> {{ $t("Color") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/COLOR/') }} </p>
+                      <p> {{ $t("Size") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/SIZE/') }} </p>
                     </ion-label>
-                    <ion-badge color="primary" slot="end">Approved</ion-badge>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("Promise date") }} </ion-label>
-                    <p slot="end">14 Jan 2021</p>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("PO arrival date") }} </ion-label>
-                    <p slot="end">14 Jan 2021</p>
-                  </ion-item>
-                  <ion-item>
-                    <ion-label> {{ $t("Last brokered") }} </ion-label>
-                    <p slot="end">California Warehouse</p>
+                    <ion-badge color="primary" slot="end"> {{ item.orderItemStatusId }} </ion-badge>
                   </ion-item>
+                  <!-- TODO: Need to handle this property -->
+                  <div v-if="item.preOrderStatus || item.backOrderStatus || item.unFillable">
+                    <ion-item>
+                      <ion-label> {{ $t("Promise date") }} </ion-label>
+                      <p slot="end"> {{ item.promisedDatetime ? $filters.formatUtcDate(item.promisedDatetime, 'YYYY-MM-DDTHH:mm:ssZ') : '-'  }} </p>
```

Comment:
Again with the dates, please ask @ymaheshwari1 what the standard we decided on was.